.github/workflows/swift.yml

```diff
@@ -7,20 +7,73 @@ on:
     branches: [ main ]
   workflow_dispatch:
 
+env:
+  PACKAGE_NAME: Typeform
+
 jobs:
   SwiftActions:
     strategy:
       matrix:
-        os: [macos-12, ubuntu-latest]
+        os: [macos-latest, ubuntu-latest]
 
     runs-on: ${{ matrix.os }}
 
+    permissions:
+      actions: read
+      checks: write
+      pull-requests: write
+
     steps:
+#   - name: Install xcbeautify (temporary)
+#     run: brew install xcbeautify
```

Comment:
@richardpiazza Is this still needed?

Reply:
Looks like the image rollout is now at 100%, so I can remove it.